staging/src/k8s.io/kubelet/pkg/cri/streaming/server_test.go

```diff
@@ -457,13 +457,15 @@ func doClientStreams(t *testing.T, prefix string, stdin io.Writer, stdout, stder
 func readExpected(t *testing.T, streamName string, r io.Reader, expected string) {
 	result := make([]byte, len(expected))
 	_, err := io.ReadAtLeast(r, result, len(expected))
+	//nolint:testifylint  // always called in go func
```

Comment:
can you please clarify that those are not go func, but rather exec. As written, it is confusing as I cannot find the go func

Reply:
/remove-lgtm